src/util/Util.ts

```diff
@@ -218,3 +218,24 @@ const durationModifiers: { [key: string]: number } = {
  * @returns A duration in seconds
  */
 export const resolveDuration: (value: string) => number = _resolve.bind(null, durationRegex, durationModifiers);
+
+// Is there no builtin type for this?
+type ObjectValueType<T> = T extends { [key: string]: infer P } ? P : never
+
+/**
+ * High-order function catching certain DiscordAPIErrors and return them, rather than rethrowing them.
+ * @param errors Error codes to catch
```

Comment:
No need to implicit the type as it's TS? I suppose you still should infer the type.

Reply:
There is no type there, that's documentation.

Reply:
[We're continuing the discussion through another platform]